runner/ansible/roles/2.1.5/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Set Test ID
```

Comment:
This is an only azure test. We could put an additional `when` condition to run in the imported roles list

Reply:
Yes, we can do that.